addon/templates/components/work-item-button.hbs

```diff
@@ -0,0 +1,19 @@
+<UkButton
+  ...attributes
+  @type={{@type}}
+  @disabled={{or @disabled @loading this.mutate.isRunning}}
+  @active={{@active}}
+  @loading={{or @loading this.mutate.isRunning}}
+  @color={{or @color "default"}}
```

Comment:
Isn't `"default"` the default color for an `UkButton`? That would make the `or` obsolete

Reply:
If we pass `@color` but it has no value it sets the color to null/ undefined overwriting the default of UkButton and having no style.